.github/workflows/tag_release.yml

```diff
@@ -0,0 +1,59 @@
+name: tag-release
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:        
+      - '**' # This includes hierarchical tags
```

Comment:
Should we filter von tags that start with a v? I.e. 'v*' maybe?

Reply:
done